src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const action = actions[i].type;
+
+    if (action === 'clear') {
```

Comment:
Use the correct action property name. Replace `if (action === 'clear')` with `if (action.type === 'clear')`